cli/Sources/TuistKit/Commands/Auth/RefreshTokenCommand.swift

```diff
@@ -0,0 +1,32 @@
+import ArgumentParser
+import Foundation
+
+struct RefreshTokenCommand: AsyncParsableCommand {
+    static var configuration: CommandConfiguration {
+        CommandConfiguration(
+            commandName: "refresh-token",
+            _superCommandName: "auth",
+            abstract: "Refreshes the token for a particular URL",
+            shouldDisplay: false
+        )
+    }
+
+    @Option(
+        name: .customLong("server-url"),
+        help: "The URL of the server the token is being refreshed for.",
+        envKey: .authRefreshTokenServerURL
+    )
+    var serverURL: String
+
+    @Option(
+        name: .customLong("lockfile-path"),
+        help: "The path to a lockfile that's used to determine if there's a refresh happening.",
+        completion: .directory,
+        envKey: .authRefreshTokenLockfilePath
+    )
+    var lockFilePath: String
```

Comment:
do we need this one to be passed? The path is conventional based on the URL, so it feels a bit unnecessary to pass it through.

Reply:
No. I removed it.